### ISSUE TITLE
BC_SUBCLASS## column name too long for shapefile

### ISSUE THREAD

**Issue by cybersea** [orcoastalmgmt/cmecs#39]
If the data get converted to shapefiles (which is inevitable...), the SUBCLASS columns will lose the identifying number at the end, since it will get truncated.    Suggest shortening the prefix so that the full field name is <= 10 characters long

**Comment by tchaddad** [comment]
Yep - field names are expected to change before we publish the final Phase II data set.
